linux_pipeline/Jenkinsfile_lis_rpm_hyperv

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env groovy
+
+properties ([
+    [$class: 'ParametersDefinitionProperty',
+        parameterDefinitions: [
+        [$class: 'StringParameterDefinition',
+            name: 'DISTRO_VERSIONS',
+            defaultValue: '',
+            description: 'Distros to be tested'],
+        [$class: 'StringParameterDefinition',
+            name: 'LIS_ARCHIVE_LINK',
+            defaultValue: '',
+            description: 'Link to the lis archive to be installed'],
+        [$class: 'StringParameterDefinition',
+            name: 'LIS_OLD_ARCHIVE_LINK',
+            defaultValue: '',
+            description: 'Link to the previous lis version']
+        ]
+    ]
+])
+
+def nodesMap = ["ws2012":"centos_6.8_32bit,centos_6.8_64bit,centos_6.9_32bit,rhel_6.7_gen2vm,rhel_6.8_x64,rhel_6.9,", \
+                "ws2012r2":"rhel_6.4_32bit,rhel_6.4_x64,rhel_6.5,rhel_6.6_x64,rhel_6.6_x32, \
+                    centos_6.4_x64,centos_6.5_x64,", \
+                "ws2016":"oracle_6.5_rhck,oracle_6.9_rhck,oracle_7.4_rhck,oracle_7.0_rhck,centos_7.0_x64, \
+                    centos_7.0_gen2vm,centos_7.2,rhel_7.0,rhel_7.1,rhel_7.2_ga,rhel_7.3_gen2vm,", \
+                "sriov":"rhel_7.3,rhel_7.4,centos_7.4,centos_7.4_x64,centos_7.3,"]
+                    
+def supportedDistros = nodesMap["ws2012"] + nodesMap["ws2012r2"] + nodesMap["ws2016"] + nodesMap["sriov"]
+
+def PowerShellWrapper (psCmd) {
+    psCmd = psCmd.replaceAll("\r", "").replaceAll("\n", "")
+    bat (script: "powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command \"\$ErrorActionPreference='Stop';[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;$psCmd;EXIT \$global:LastExitCode\"",returnStatus: true)
```

Comment:
make the powershell wrapper return the value returned by bat, right now the change is useless